api/turing/config/config.go

```diff
@@ -231,6 +231,16 @@ type DeploymentConfig struct {
 	MaxMemory                 Quantity      `validate:"required"`
 	MaxAllowedReplica         int           `validate:"required"`
 	TopologySpreadConstraints []corev1.TopologySpreadConstraint
+	PodDisruptionBudget       PodDisruptionBudgetConfig
+}
+
+// PodDisruptionBudgetConfig are the configuration for PodDisruptionBudgetConfig for
+// Turing services.
+type PodDisruptionBudgetConfig struct {
+	Enabled bool
+	// Can specify only one of maxUnavailable and minAvailable
+	MaxUnavailable string
```

Comment:
Shall we instead name this as `MaxUnavailablePercent` and use `*int` ? Or, if we are sticking with the string type, should we check that the value is of the expected format (like "20%")? Would be nice to add some checks around this as well because otherwise, the earlier we can discover a problem with this config is when we deploy a router, which may be too late.